documentation/content/el/books/handbook/desktop/_index.adoc

```diff
@@ -340,7 +340,7 @@ endif::[]
 
 [source,shell]
 ....
-# pkg_add -r koffice-kde4
+# pkg install koffice-kde4
```

Comment:
There is no longer any port with `koffice` in its name.

Reply:
calligra does https://www.freshports.org/editors/calligra/